components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -514,6 +515,104 @@ bool PublisherInfoDatabase::InsertOrUpdateActivityInfos(
   return transaction.Commit();
 }
 
+uint64_t PublisherInfoDatabase::GetLastReconcileDate() {
```

Comment:
I don't think that you need a separate method for this. If you have a LedgerClient::DeleteRewardsData(data types enum) you can just make it part of the delete statement

Reply:
This isn't used for deletion. This is to display on the Clear Rewards UI Auto Contribute section the date of the last contribution (if there was one).